sbt-scalajs-bundler/src/main/scala/scalajsbundler/PackageJson.scala

```diff
@@ -51,6 +51,9 @@ object PackageJson {
       JSON.obj(
         (
           additionalNpmConfig.toSeq :+
+          "private" -> JSON.bool(true) :+
+          "license" -> JSON.str("UNLICENSED") :+
+          "description" -> JSON.str("") :+
```

Comment:
Ok fixed. Also, I'm leaving everything as separate commits so some of these simple changes we can pull out and PR separately if needed.